packages/legend-data-cube/src/stores/core/__tests__/DataCubeQuerySnapshotBuilder.repl-test.ts

```diff
@@ -103,7 +103,7 @@ const cases: BaseSnapshotAnalysisTestCase[] = [
     'Invalid: Unsupported function composition: select()->filter()',
     'select(~a)->filter(x|$x.a==1)',
     ['a:Integer'],
-    'Unsupported function composition select()->filter() (supported composition: extend()->filter()->select()->[sort()->pivot()->cast()]->[groupBy()->sort()]->extend()->sort()->limit())',
+    '',
```

Comment:
also, please rename this test to use the new extension `data-cube-test.ts`

Reply:
done